src/frontend/js/contexts/SessionContext/JoanieSessionProvider.tsx

```diff
@@ -110,11 +113,21 @@ const JoanieSessionProvider = ({ children }: React.PropsWithChildren<{}>) => {
     () => ({
       user,
       isPending,
+      joanieUserProfile: joanieUserProfile.item,
+      isPendingJoanieUserProfile: joanieUserProfile.states.fetching,
```

Comment:
There is a hook dedicated to this information, let's use it where it is needed WDYT ?

Reply:
@jbpenrath should i also remove `isPending` as it's not used in our code ?

Reply:
Actually, we know we are fetching user information as long as user is `undefined`.
So indeed we could remove this indicator, but as this is a little out of the scope of this PR, I'll do nothing